### DISCUSSION TITLE
integration with moti

### DISCUSSION THREAD

**Discussion by brunowilliang** [GeekyAnts/NativeBase#5069] [General]
### Description

It would be interesting to include standard animations in the View, Box and presseable

### Problem Statement

It would be interesting to include standard animations in the View, Box and presseable

### Proposed Solution or API

It would be interesting to include standard animations in the View, Box and presseable

**Comment by Krithikj20** [comment]
@Brunowilliang We appreciate your feedback. We will pass it to the concerned team so that they check the possibilities of adding the same in future updates. Thank you.

**Comment by Viraj-10** [comment]
Hi @Brunowilliang, We have a `Transition` component for simple animations with a recipe of Fade, SlideFade, ScaleFade etc. So Users can easily create simple animations. Adding extra prop for animation will require component level computation which affects the component's performance. Box, Pressable, and View are primitive components from which composite components are created so it affects a lot more components. 
Although we are open to discussion. let us know. What's your take on this.
I am moving this to the discussion section.

**Comment by brunowilliang** [comment]
Hi @Viraj-10, could create a transition docs in buttons!

**Reply by sanna231** [reply]
We are writing an article that will include animation, NativeBase integration with Moti, and reanimated.
